train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java

```diff
@@ -18,6 +19,11 @@ public TrainControllerImpl() {
 		TimerTask periodicSpeedControlTask = new TimerTask() {
 			@Override
 			public void run() {
+
+				Random rand = new Random();
+
+				step = rand.nextInt(20);
```

Comment:
We have an overrideJoystickPosition method for overriding joystick movement, please use that for simualtion purposes too.